testSuite/scripts/test_upload_page_blob.py

```diff
@@ -66,7 +66,7 @@ def test_page_blob_upload_partial_sparse_file(self):
         # execute azcopy pageblob upload.
         destination_sas = util.get_resource_sas(file_name)
         result = util.Command("copy").add_arguments(file_path).add_arguments(destination_sas).add_flags("log-level", "info"). \
-            add_flags("block-size", "4194304").add_flags("blob-type","PageBlob").execute_azcopy_copy_command()
```

Comment:
wow that's a lot of manual changes.. we should definitely improve this later

Reply:
Agreed!  But not doing anything about it now ;-)

Reply:
:shipit: